images/packages/binaries/libgcrypt/werf.inc.yaml

```
@@ -0,0 +1,62 @@
+---
+image: {{ $.ImageType }}/{{ $.ImageName }}
+final: false
+fromImage: builder/scratch
+import:
+- image: {{ $.ImageType }}/{{ $.ImageName }}-builder
+  add: /out
+  to: /libcrypt
+  before: setup
+
+---
+{{- $version := "libgcrypt-1.10.2" }}
```

Use this to get the version from a file `component_versions/version_map.yml`
```suggestion
{{- $version := get $.Package $.ImageName }}
```